### DISCUSSION TITLE
Page: Disable the scroll up (or down)

### DISCUSSION THREAD

**Discussion by HackTechDev** [rexrainbow/phaser3-rex-notes#300] [Q&A] [answered]
Hi !
For this example : 
https://codepen.io/rexrainbow/pen/vPWzBa
How to disable the scroll up when I click on the page (I don't release the mouse) and move to bottom?
Thank you.

**Comment by rexrainbow** [comment, marked as answer]
[Disable scroller or slider bar](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ui-scrollablepanel/#enabledisable-scrolling)

```js
panel.setScrollerEnable(false);
panel.setSliderEnable(false);
```
